### ISSUE TITLE
Add method in Api module to move keyboard focus

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#8212]
Just started learning addon development, and am surprised there isn't a built-in way to do this, as this is a common task.

**Comment by Neurrone** [comment]
@Brian1Gaff for addon development, e.g pressing a hotkey to move focus to another part of the application.

**Comment by Adriani90** [comment]
@josephsl since you maintained golden cursors for a long time, this is moving the mouse cursor but still it is focus related. Do you think this request is relevant?
@Neurrone can you please elaborate more on what exactly you want to achieve?

**Comment by Neurrone** [comment]
As the issue title says, there isn't a way to move keyboard focus to a specific element that is able to gain focus. This is useful if there is no native shortcut to get to a specific part of an application. For example, pressing f6 to cycle between different parts of the application.

**Comment by Adriani90** [comment]
cc: @nvdaes, @CyrilleB79 since you have much experience with add-ons, is the documentation lacking this aspect? Or is there really no way for add-ons to move keyboard focus?
